add-on/chrome/bootstrap.js

```diff
@@ -681,28 +677,57 @@ var WindowListener = {
         return str;
       },
 
+      _browserSharePaused: false,
```

Comment:
Actually, rather than making this a member variable, make it locally scoped to _maybeShowBrowserSharingInfoBar using let.